selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/stack/CreateWorkspaceFromEclipseVertxStackTest.java

```diff
@@ -0,0 +1,114 @@
+import org.eclipse.che.selenium.pageobject.dashboard.CreateWorkspaceHelper;
+import org.eclipse.che.selenium.pageobject.dashboard.Dashboard;
+import org.openqa.selenium.By;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+/** @author Skoryk Serhii */
+public class CreateWorkspaceFromEclipseVertxStackTest {
+
+  private static final String WORKSPACE_NAME = generate("workspace", 4);
+  private static final String VERTX_HEALTH_CHECKS_BOOSTER = "vertx-health-checks-booster";
+  private static final String VERTX_HTTP_BOOSTER = "vertx-http-booster";
+
+  private List<String> projects = ImmutableList.of(VERTX_HEALTH_CHECKS_BOOSTER, VERTX_HTTP_BOOSTER);
+
+  @Inject private Ide ide;
+  @Inject private Consoles consoles;
+  @Inject private Dashboard dashboard;
+  @Inject private DefaultTestUser defaultTestUser;
+  @Inject private ProjectExplorer projectExplorer;
+  @Inject private CreateWorkspaceHelper createWorkspaceHelper;
+  @Inject private TestWorkspaceServiceClient workspaceServiceClient;
+
+  @BeforeClass
+  public void setUp() {
+    dashboard.open();
+  }
+
+  @AfterClass
+  public void tearDown() throws Exception {
+    workspaceServiceClient.delete(WORKSPACE_NAME, defaultTestUser.getName());
+  }
+
+  @Test
+  public void checkWorkspaceCreationFromEclipseVertxStack() {
+    createWorkspaceHelper.createWorkspaceFromStackWithProjects(
+        ECLIPSE_VERTX, WORKSPACE_NAME, projects);
+
+    ide.switchToIdeAndWaitWorkspaceIsReadyToUse();
+
+    projectExplorer.waitProjectInitialization(VERTX_HEALTH_CHECKS_BOOSTER);
+    projectExplorer.waitProjectInitialization(VERTX_HTTP_BOOSTER);
+  }
+
+  @Test(priority = 1)
+  public void checkVertxHealthChecksBoosterProjectCommands() {
+    consoles.executeCommandFromProjectExplorer(
+        VERTX_HEALTH_CHECKS_BOOSTER, BUILD_GOAL, BUILD_COMMAND, BUILD_SUCCESS);
+
+    consoles.executeCommandFromProjectExplorer(
+        VERTX_HEALTH_CHECKS_BOOSTER,
+        RUN_GOAL,
+        RUN_COMMAND,
+        "[INFO] INFO: Succeeded in deploying verticle");
+    consoles.checkWebElementVisibilityAtPreviewPage(By.id("_vert_x_health_check_booster"));
+    consoles.closeProcessTabWithAskDialog(RUN_COMMAND);
+
+    consoles.executeCommandFromProcessesArea(
+        "dev-machine",
+        DEBUG_GOAL,
+        DEBUG_COMMAND,
+        "[INFO] Listening for transport dt_socket at address: 5005");
+    consoles.closeProcessTabWithAskDialog(DEBUG_COMMAND);
+  }
+
+  @Test(priority = 1)
+  public void checkVertxHttpBoosterProjectCommands() {
+    consoles.executeCommandFromProjectExplorer(
+        VERTX_HTTP_BOOSTER, BUILD_GOAL, BUILD_COMMAND, BUILD_SUCCESS);
+
+    consoles.executeCommandFromProjectExplorer(
+        VERTX_HTTP_BOOSTER, RUN_GOAL, RUN_COMMAND, "[INFO] INFO: Succeeded in deploying verticle");
+    consoles.checkWebElementVisibilityAtPreviewPage(By.id("_vert_x_health_check_booster"));
```

Comment:
It's interesting that here we are checking on `_vert_x_health_check_booster` while the project is about _http_ boosting.